doc/Makefile

```diff
@@ -4,7 +4,7 @@
 # You can set these variables from the command line.
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = ibex
+SPHINXPROJ    = cve2
```

Comment:
Hmm.  That _might_ break the documentation build.  We can accept the risk for now.